BuildAll.bat

```diff
@@ -9,6 +9,9 @@ if ERRORLEVEL 1 (goto error)
 dotnet build Olive.Audit
 if ERRORLEVEL 1 (goto error)
 
+dotnet build Services\Olive.Encryption
```

Comment:
this project was missing from the build process and it was a dependancy to the next one.